src/library/modules/Kcsapi.js

```diff
@@ -1454,20 +1453,17 @@ Previously known as "Reactor"
 							console.info('Reported new API Gauge Type',eventData.api_gauge_type);
 					}
 				}
-
+				
 				// Check default gauge info
-				if(typeof maps[key] !== 'undefined') {
-					if(typeof maps[key].dkind === 'undefined') {
-						maps[key].dkind = maps[key].kind;
-					}
+				if(typeof maps[key].dkind === 'undefined') {
```

Comment:
I'll push for 28.2 tonight to bring this back along with any fix you guys want to include